tutorials/raw/plot_events.py

```diff
@@ -0,0 +1,153 @@
+data, and cropping it to save memory:
+"""
+
+import os
+import numpy as np
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, verbose=False)
+raw.crop(tmax=60).load_data()
+
+###############################################################################
+# The tutorial :doc:`../intro/plot_object_annotations` describes in detail how
+# to obtain an Events array from either :term:`STIM channels <stim channel>` or
+# embedded event arrays within a raw data file, so we won't discuss that in
+# detail here. As a quick reminder, the :ref:`sample dataset <sample-dataset>`
+# includes experimental events recorded on channel ``STI 014``. Here parse the
+# events from that channel using :func:`mne.find_events`:
+
+events = mne.find_events(raw, stim_channel='STI 014')
+
+###############################################################################
+# Reading and writing events to file
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Event arrays are :class:`NumPy array <numpy.ndarray>` objects, so they could
+# be saved to disk as binary :file:`.npy` files using :func:`numpy.save`.
+# However, MNE-Python provides convenience functions :func:`mne.read_events`
+# and :func:`mne.write_events` for reading and writing event arrays as either
+# text files (common file extensions are :file:`.eve`, :file:`.lst`, and
+# :file:`.txt`) or binary :file:`.fif` files. The example dataset includes the
+# results of ``mne.find_events(raw)`` in a :file:`.fif` file. Since we've
+# truncated our :class:`~mne.io.Raw` object, it will have fewer events than the
+# events file loaded from disk (which contains events for the entire
+# recording), but the events should match for the first 60 seconds anyway:
+
+sample_data_events_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                       'sample_audvis_raw-eve.fif')
+events_from_file = mne.read_events(sample_data_events_file)
+assert np.array_equal(events, events_from_file[:len(events)])
+
+###############################################################################
+# When writing event arrays to disk, the format will be inferred from the file
+# extension you provide. By convention, MNE-Python expects events files to
+# either have an :file:`.eve` extension or to have a file basename ending in
+# ``-eve`` or ``_eve`` (e.g., :file:`{my_experiment}_eve.fif`), and will issue
+# a warning if this convention is not respected.
+#
+#
+# Subselecting and combining events
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# The output of :func:`~mne.find_events` above (copied here) told us the number
+# of events that were found, and the unique integer event IDs present:
+#
+# .. code-block:: none
+#
+#     86 events found
+#     Event IDs: [ 1  2  3  4  5 32]
```

Comment:
I would put this explanation in the new section `Obtaining events from a raw object` and here only talk about `pick_events`

Reply:
If this has to stay here (which is where I like the most) and we still need `events = mne.find_events(bla)`, on the top for `Reading and writing events from a file` then instead of just copying the output to make the point, I would re-do the commant. Like this when I'm following the tutorial, I can copy paste without having to move up and down. Most likely If I'm reading here I don't know from the top of the head how should I call `find_events` to get this output.